### DISCUSSION TITLE
下载服务端 jar 文件

### DISCUSSION THREAD

**Discussion by kongxia114** [Meloong-Git/PCL#1450] [提问] [answered]
一键 下载服务端 jar 文件
MOD 端
官方端
插件端

**Comment by WForst-Breeze** [comment, marked as answer]
以内容已被 #112 提出，您可以到 #113 为其投票！
btw，PCL2 不会支持除原版服务端以外的**服务端**核心（eg：spigot、paper、fabric、forge 等）下载，这超出了 PCL2 的范畴了

**Comment by Yue-cn** [comment]
若需要，直接调用api下载即可。

**Comment by Chiloven945** [comment]
但是他是启动器（

**Comment by Sniper2711** [comment]
MOD端可以：下载-手动安装包-Forge/Fabric对应版本安装器-Install Server-选择目录-安装

**Reply by Sniper2711** [reply]
其他的……再说吧
